documentation/data/4_Risks.md

```diff
@@ -0,0 +1,25 @@
+# Risks
+
+This document aims to outline risks which have been identified as likely to impact the operation of the FBIT service.
+
+## Identified Risks
+
+### Changing Data Schemas
+
+It has been seen in previous year's data releases that the format of the various data sources changes regularly. This may include, but is not limited to:
+* Column names changing.
+* New Columns being added to the data files.
+* Columns being removed from the data files, likely due to a redundancy in the metric. (e.g. Covid Funding is now no longer considered in the service post 2022)
+* The summation logic for different columns may change. 

```

Comment:
Assume you mean here like `Total Expenditure  E01 to E32` - how do we provide and validate year and year that this is actually the sum of E01:E32 excluding E30?

Reply:
Exactly that. We could put some testing in to check our data for this. I can add it as a #TODO for myself and Faizi?